### ISSUE TITLE
`Unable to open 'Settings'` error after running `configure extension settings` for a newly installed extension

### ISSUE THREAD

**Issue by mjbvz** [Microsoft/vscode#72586]
Issue Type: <b>Bug</b>

**Repo**
1. Install a theme extension in the editor (such as [this](https://marketplace.visualstudio.com/items?itemName=gerane.Theme-HotDogStand))
1. Without restarting, in the extensions viewlet click on the gear for the newly installed extension and select `configure extension settings` 

**Bug**
See `Unable to open 'Settings' ...` message

In the dev tools, see:

```
/Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:2362 Unable to open 'Settings': Cannot read property 'element' of undefined.
onDidNotificationChange @ /Applications/Visual Studio Code - Insiders.app/Contents/Resources/app/out/vs/workbench/workbench.main.js:2362
```

VS Code version: Code - Insiders 1.34.0-insider (f89390fcec44a24ce74dfc4efbc1438f35b7357a, 2019-04-18T13:31:41.962Z)
OS version: Darwin x64 18.5.0

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-4770HQ CPU @ 2.20GHz (8 x 2200)|
|GPU Status|2d_canvas: enabled<br>checker_imaging: disabled_off<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: enabled<br>rasterization: enabled<br>surface_synchronization: enabled_on<br>video_decode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|2, 3, 3|
|Memory (System)|16.00GB (0.13GB free)|
|Process Argv||
|Screen Reader|no|
|VM|29%|

**Comment by georgejiang97** [comment]
Hey, I have it working on my computer, but it is a change to "vscode/out/vs/workbench/contrib/preferences/electron-browser/settingsEditor2.js:846:61", which is part of the .gitignore of this repo - are there other files that I could look at editing for this bug?

**Comment by roblourens** [comment]
Dupe of https://github.com/Microsoft/vscode/issues/72354

**Comment by mjbvz** [comment]
Hey @roblourens, there's also the second exception in the dev console about `Unable to open 'Settings': Cannot read property 'element' of undefined` so do think this is a duplicate of #72354. I've edited my issue to make it more clear that this was the main issue I was seeing.

Feel free to close again if that is not correct

**Comment by roblourens** [comment]
I thought this was fixed with my fix for #72354 but I see that it's not fixed in today's build...

**Comment by roblourens** [comment]
Actually, I can't repro this anymore. I think my previous change was missing from yesterday's first build. Let me know if you still see it.
